src/main/java/org/colonelkai/application/Settings.java

```diff
@@ -1,12 +1,15 @@
 package org.colonelkai.application;
 
+import org.colonelkai.ForwardLauncher;
+
 import java.nio.file.Path;
 import java.util.HashMap;
 
 public class Settings {
     // all of the settings :)
-    private boolean isDarkMode;
-    private Path defaultInstallFolder;
+    // default values?
+    private boolean isDarkMode = false;
```

Comment:
Primitive type (the class types that show in a different colour) all have default values. Boolean's default is false.